index.html

```diff
@@ -0,0 +1,275 @@
+                </div>
+    
+                <div class="tags">
+                    <div>
+                        <li class="tag"><a href="">html</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">CSS</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">Javascript</a></li>
+                    </div>
+                </div>
+    
+                <div class="action">
+                    <div class="btn-action">
+                        <a class="see">See Project</a>
+                    </div>
+                </div>
+            </div>
+    
+    
+        </section>
+
+
+        <section class="about">
+            <div class="myself">About Myself</div>
+            <p class="content"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <div class="connect"><a href="#">LET'S CONNECT</a> </div>
+            <div class="social">
+                <ul class="social_nav">
+                    <li><a href="twitter.com/tawakulikhadim"><img src="assets/image_social/vitctor_t.svg" alt="Twitter"></a>
+                    </li>
+                    <li><a href="https://github.com/tawakuliKH"><img src="assets/image_social/victor_g.svg"
+                                alt="Github"></a></li>
+                    <li><a href=""><img src="assets/image_social/victor_h.svg" alt=""></a></li>
+                    <li><a href=""><img src="assets/image_social/victor_m.svg" alt="Mediam"></a></li>
+                    <li><a href=""><img src="assets/image_social/victor_l.svg" alt="Linked In"></a></li>
+                </ul>
+    
+            </div>
+            <div class="action-resume">
+                    <a class="resume">Get MY Resume</a>
+            </div>
+
+            <div class="skills">
+                <div>
+                    <li class="first lang"><a href="#" class="languages">Languages</a><a href=""><img src="assets/Icon - Chevron.png" alt=""></a> </li>
+                    <li><a href="#"><img src="assets/css.png" alt=""></a></li>
+                    <li><a href="#"><img src="assets/html.png" alt=""></a></li>
+                    <li><a href="#"><img src="assets/js.png" alt=""></a></li>
+                    <li class="first icons"><a href="#" class="languages">Frameworks</a><a href=""><img src="assets/Icon - Chevron - Right - Larger.png" alt=""></a> </li>
+                    <li class=""><img src="assets/Separator Bottom.png" alt=""></li>
+                    <li class="first icons"><a href="#" class="languages">Skills</a><a href=""><img src="assets/Icon - Chevron - Right - Larger.png" alt=""></a> </li>
+                    <li class=""><img src="assets/Separator Bottom.png" alt=""></li>
+
```

Comment:
Kindly remove extra spaces in basecode, extra spaces and newlinex makes your code not clean.